tests/subprograms/test_challenge_121.py

```diff
@@ -0,0 +1,63 @@
+import pytest
+
+from exercises.subprograms import challenge_121
+
+
+test_data_delete = [
+    (["name1", "name2", "name3"], 1, ["name1", "name3"]),
+    (["name1", "name2", "name3"], 2, ["name1", "name2"]),
+]
+
+test_data_delete_error = [
```

Comment:
Using parametrize only makes sense if you have more than one set of parameters. Can you rewrite these tests without the extra overhead, please?